src/pages/schedule.js

```diff
@@ -0,0 +1,365 @@
+import React from 'react';
+const moment = require('moment-timezone');
+import Announcement from '../components/announcement';
+
+import './schedule.css';
+
+export default class Schedule extends React.Component {
+  state = {}
+
+  componentDidMount() {
+    this.updateHandle = setInterval(this.updateTime, 1000 * 60)
```

Comment:
Not sure how this will work with the static build?

Reply:
The static build will not mount the component, so this only happens in the frontend :)